recipes/xeus-octave/build.sh

```diff
@@ -0,0 +1,6 @@
+
+#!/bin/bash
+
+cmake -DCMAKE_INSTALL_PREFIX=$PREFIX
```

Comment:
We supply a set of sane defaults here that you can use instead.
```suggestion
cmake ${CMAKE_ARGS}
```

Reply:
Hum! Didn't know about that, thank you.